dbaas/psmdb/config.go

```diff
@@ -522,7 +523,13 @@ func (cr *PerconaServerMongoDB) SetNew(clusterName, rsName string, f *pflag.Flag
 	cr.Spec.Replsets = []*ReplsetSpec{
 		rs,
 	}
-
+	if p == dbaas.Minishift || p == dbaas.Minikube {
+		none := "none"
+		for i, _ := range cr.Spec.Replsets {
+			cr.Spec.Replsets[i].Resources = nil
```

Comment:
Did you check this code? I believe it'll cause an operator error because of erased resource requests for PVC.